src/features/unified-share-modal/__tests__/UnifiedShareForm.test.js

```diff
@@ -567,6 +568,7 @@ describe('features/unified-share-modal/UnifiedShareForm', () => {
 
         test('should return true if new shared link and link is loaded', () => {
```

Comment:
can you update this test case description to include that createSharedLinkOnLoad is true here?

Reply:
can you also add a test case that is similar to this one, but without `createSharedLinkOnLoad: true`? this will help document the expected behavior for just `focusSharedLinkOnLoad: false`